api/swagger.yml

```diff
@@ -610,9 +610,13 @@ components:
           type: string
           enum: [initialized, not_initialized, comm_prefs_done]
         oidc_enabled:
+          deprecated: true
           type: boolean
+          description: "Deprecated: embedded OIDC support is now deprecated"
```

Comment:
Consider adding "... and will be removed in the upcoming versions"

Reply:
Not done: this is a product decision.  Our [communication on CHANGELOG](https://github.com/treeverse/lakeFS/blob/bf692ea20db7e9f9f48ef646a3eb36afbed53c33/CHANGELOG.md?plain=1#L7) is merely "deprecated".

Reply:
If we expect OIDC to remain in lakeFS cloud and for lakeFS cloud to be compatible with existing clients, I would suggest not marking it as deprecated in the API itself (docs changes that were already pushed should clarify and suffice)

Reply:
1. It is hard to imagine a client using getSetupState _and_ a generated OpenAPI client.  webui does use the endpoint, but needs to change and anyway doesn't use a generated client.
1. If there is such a client, it probably doesn't use `oidc_enabled` -- and will indeed be unaffected by this change.
1. If there is such a client that _does_ use `oidc_enabled` then it will stop working, because this property is indeed going away in lakeFS and also in Cloud.

Reply:
Thanks for the context. I agree it should be marked as deprecated then.